split_expenses_ui/src/components/ProcessExpense/ProcessExpense.jsx

```diff
@@ -90,11 +91,11 @@ function ProcessExpense(props) {
                                       name={product.productName}
```

Comment:
Thanks for the changes. Shall we make the product name also consistent? Shall we use either camel case or '_' everywhere?

Reply:
Sure. Actually the api is  invoice captured details from back to front. 

Is it possible to parse the products data in back-end instead of doing it in the front side.?
In that way we can minimise the loading time in the front application.

Reply:
Updated the changes in the latest PR. Thanks for the suggesting changes

Reply:
@gsundeep-tech  I have integrated the back-end changes. Could you please validate it again?

Thanks.